### PR TITLE
docs: fix some typos

### PR DESCRIPTION
**Pull request by joergsteffens**
(cherry picked from commit 8a5aca85559caba9258d081b081f0aba287f3bef)

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [ ] ~Separate commit for this PR in the CHANGELOG.md, PR number referenced is same~
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [ ] ~Source code changes are understandable~
- [ ] ~Variable and function names are meaningful~
- [ ] ~Code comments are correct (logically and spelling)~
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 1]
```diff
@@ -183,20 +183,20 @@ API mode 1 (on).
...
-or (form times where they have only been one API flavour)
+or (from times where they was only one API flavour)
 
 ::
 
     .api
 
-This mode is intended to create output that is earlier parsable.
-Internaly some commands vary there output for the API mode 1, but not
+This mode is intended to create output that is earlier to parse.
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:192]:
```suggestion
This mode is intended to create output that is easier to parse.
```

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 2]
```diff
@@ -183,20 +183,20 @@ API mode 1 (on).
...
+Internaly some commands vary their output for API mode 1, but not
 all.
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:194]:
```suggestion
Internaly some commands vary their output for API mode 1.
```

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 3]
```diff
@@ -183,20 +183,20 @@ API mode 1 (on).
...
 
-In API mode 1 some output is only delimted by the end of a packet, by
-not a new line. bconsole does not display end of packets (for good
-reason, as some output (e.g. :bcommand:`status`) is send in multiple packets).
-If running in a bconsole, this leads not parsable output for human.
+In API mode 1 some output is only delimited by the end of a packet, not
+by a new line. bconsole does not display the end of packets (for good
+reason, as some output (e.g. :bcommand:`status`) is sent in multiple packets).
+If running in a bconsole, this leads to unparsable output for human.
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:199]:
```suggestion
In API mode 1 some output is delimited by end of a packet and not by a newline character.
Thus the output of API mode 1 may be non-parseable when being used in bconsole, as end of packet is not displayed there.
```

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 4]
```diff
@@ -237,9 +237,9 @@ differs from API mode 1 in several aspects:
    make it easier to implement a full JSON-RPC service later.
 -  No user interaction inside a command (meaning: if not all parameter
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:238]:
```suggestion
-  No user interaction inside a command (meaning: if not all parameters
```

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 5]
```diff
@@ -237,9 +237,9 @@ differs from API mode 1 in several aspects:
...
    are given to a :bcommand:`run` command, the command fails).
--  Each command creates exaclty one responce object.
+-  Each command creates exactly one responce object.
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:240]:
```suggestion
-  Each command creates exactly one response object.
```

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 6]
```diff
@@ -237,9 +237,9 @@ differs from API mode 1 in several aspects:
...
 
-Currently a subset of the available commands return there result in JSON
+Currently a subset of the available commands return their result in JSON
 format, while others still write plain text output. When finished, it
 should be safe to run all commands in JSON mode.
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:244]:
```suggestion
format, while others still write plain text output.
```

File: docs/manuals/source/DeveloperGuide/api.rst [hunk 7]
```diff
@@ -441,7 +441,7 @@ REST API
 
 Introduction
 ~~~~~~~~~~~~
-This is an experimental and preliminary backend for development purposes. The endpoints are subject for review and maybe changed in the future.
+This is an experimental and preliminary backend for development purposes. The endpoints are subject to review and may be changed in the future.
```
Comment (arogge) [inline docs/manuals/source/DeveloperGuide/api.rst:444]:
```suggestion
This is an experimental and preliminary backend for development purposes. The endpoints are subject to review and may change in the future.
```

**Review by arogge — CHANGES_REQUESTED** [review]
I had some additions you might want to consider.

File: docs/manuals/source/DeveloperGuide/api.rst (hunk 3 above)
Comment (joergsteffens) [reply docs/manuals/source/DeveloperGuide/api.rst:198]: If I remember correctly, the output is displayed. It is just very hard to read, as line-breaks are missing.

**Review by arogge — APPROVED** [review]

**Pull request merged**